src/main/java/ai/deepcode/javaclient/core/AnalysisDataBase.java

```diff
@@ -80,7 +78,7 @@ protected AnalysisDataBase(
     }
     if (!brokenKeys.isEmpty()) {
       dcLogger.logWarn(
-          "Suggestions not found for " + brokenKeys.size() + " files: " + brokenKeys.toString());
+          "Suggestions not found for " + brokenKeys.size() + " files: " + brokenKeys);
```

Comment:
Will `brokenKeys` be logged as hash to the object or does logger do the job to nicely print arraylist out?

Reply:
afaik in Java casting any Object to string will cause `.toString()` to be called by default.
PS have no idea why I did put it there before...

Reply:
It's not a change for the API, just some cleaning up of unnecessary calls. As it's Java, it's probably a hash, but I would not change stuff that was there before to keep scope in check.